src/routes/learn/getting-started-with-solid/why-solid.md

```diff
@@ -0,0 +1,16 @@
+<title>Why Solid?</title>
+
+- Discussion of traditional, server-rendered apps (MPAs)
```

Comment:
Maybe too much info? Not sure how much we should explain the motivation for SPAs and UI frameworks in general.

Reply:
I would leave out the MPA / SPA discussion and include it in the "build an app with solid" tutorial. My assumption is that people are coming from HTML/CSS/JS knowledge, so building off from that seems to be an ideal starting point.

It's an interesting idea to show hitting an endpoint with Vanilla JS. That leads great into createResource. I'm not sure if it should be the motivating example from the get-go: my thoughts are we want something with a lot of user interaction, so we can show how imperatively managing state leads to spaghetti code